components/button/button-subtle.js

```diff
@@ -70,36 +72,31 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 					background-color: transparent;
 					border-color: transparent;
 					font-family: inherit;
-					padding: 0 var(--d2l-button-subtle-padding-inline-end) 0 var(--d2l-button-subtle-padding-inline-start);
+					padding-bottom: 0;
+					padding-inline-start: var(--d2l-button-subtle-padding-inline-start);
+					padding-inline-end: var(--d2l-button-subtle-padding-inline-end);
+					padding-top: 0;
 					position: relative;
 				}
 
-				:host([dir="rtl"]) button {
-					padding: 0 var(--d2l-button-subtle-padding-inline-start) 0 var(--d2l-button-subtle-padding-inline-end);
-				}
-
 				:host([slim]) button {
 					--d2l-button-subtle-padding-inline-start: 0.5rem;
 					--d2l-button-subtle-padding-inline-end: 0.5rem;
 					min-height: 1.5rem;
 				}
 
-				:host([slim][icon]) button {
+				:host([slim]) button.d2l-button-subtle-has-icon {
 					--d2l-button-subtle-padding-inline-start: 0.4rem;
 					--d2l-button-subtle-padding-inline-end: 0.5rem;
 				}
 
-				:host([slim][icon][icon-right]) button {
+				:host([slim][icon-right]) button.d2l-button-subtle-has-icon {
 					--d2l-button-subtle-padding-inline-start: 0.5rem;
 					--d2l-button-subtle-padding-inline-end: 0.4rem;
 				}
 
 				:host([h-align="text"]) button {
-					left: calc(var(--d2l-button-subtle-padding-inline-start) * -1);
-				}
-				:host([dir="rtl"][h-align="text"]) button {
-					left: 0;
-					right: calc(var(--d2l-button-subtle-padding-inline-start) * -1);
+					inset-inline: calc(var(--d2l-button-subtle-padding-inline-start) * -1) 0;
```

Comment:
`inset-inline-start` is probably more analogous to `left` and would mean you don't need to have the `0` value? 🤷 

```suggestion
					inset-inline-start: calc(var(--d2l-button-subtle-padding-inline-start) * -1);
```